src/overlays/actors/ovl_Arrow_Fire/z_arrow_fire.c

```diff
@@ -62,6 +62,8 @@ void ArrowFire_Charge(ArrowFire* this, GlobalContext* globalCtx) {
     EnArrow* arrow;
 
     arrow = (EnArrow*)this->actor.parent;
+
+    // if arrow has been put away, kill the fire effect
```

Comment:
The word "effect" usually refers to the effects system (particle system) of the game (like various kinds of dust, sparks...) (files in `src/overlays/effects/`, kinda similar to actors), so we try to avoid using it in other contexts

I would suggest if I may
```suggestion
    // if arrow has been put away, also remove the fire part
```